netlify/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+if [[ -z "${GITHUB_ACTIONS}" ]]; then
+    echo "Running in netlify CI or manual"
+    yarn build:netlify
```

Comment:
```suggestion
    exec yarn build:netlify
```

To return exit codes